selendroid-server/src/main/java/io/selendroid/server/model/DefaultAppiumUiAutomatorDriver.java

```diff
@@ -0,0 +1,279 @@
+    }
+
+    @Override
+    public byte[] takeScreenshot() {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public Keyboard getKeyboard() {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public String getTitle() {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public void get(String url) {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public TouchScreen getTouch() {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public void addCookie(String url, Cookie cookie) {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public void deleteCookie(String url) {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public void deleteNamedCookie(String url, String name) {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public Set<Cookie> getCookies(String url) {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public Object executeScript(String script, JSONArray args) {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public Object executeScript(String script, Object... args) {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public Object executeAsyncScript(String script, JSONArray args) {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public String getContext() {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public Set<String> getContexts() {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public Dimension getWindowSize() {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public void setFrameContext(Object index) throws JSONException {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public void back() {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public void forward() {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public void refresh() {
+        throw new NoSuchMethodImplementationException("Method not implemented");
+    }
+
+    @Override
+    public boolean isAlertPresent() {
+        return false;
```

Comment:
this is not a correct implementation of the method, selendroid has actual logic that detects alert presence.